Kickstarter-iOS/Views/Cells/SettingsNotificationCell.swift

```diff
@@ -71,10 +74,11 @@ final class SettingsNotificationCell: UITableViewCell, NibLoading, ValueCell {
       |> UIButton.lens.image(for: .selected) .~ Library.image(named: "email-icon",
                                                       tintColor: .ksr_green_700,
                                                       inBundle: Bundle.framework)
-      |> UIButton.lens.accessibilityLabel %~ { _ in Strings.Email_notifications() }
 
     _ = self.pushNotificationsButton
       |> notificationButtonStyle
+      |> \.accessibilityTraits .~ .none
+      |> UIButton.lens.accessibilityHint %~ { _ in Strings.Double_tap_to_toggle_setting() }
```

Comment:
Since the `accessibilityHint` string is the same for both push and email buttons, I would consider moving this line to the `notificationButtonStyle`, avoiding to repeat ourselves (The same would apply for `\.accessibilityTraits .~ .none`).

Reply:
Do we include strings in any of our other style functions?

Reply:
The `settingsLogoutButtonStyle` includes strings

Reply:
@justinswart We do!